components/common/back_button.tsx

```diff
@@ -38,7 +43,7 @@ export default class BackButton extends React.PureComponent {
                         id='generic_icons.back'
                         defaultMessage='Back Icon'
                     >
-                        {(title) => (
+                        {(title: any) => (
```

Comment:
Title here should be always a string

Reply:
In that case, I'll have to do something like this:
```tsx
{(...title: Array<string | JSX.Element>) => (
    <span
        id='back_button_icon'
        className='fa fa-1x fa-angle-left'
        title={title[0].toString()}
    />
)}
```
which looks pretty ugly. What do you say?

Reply:
Can't it be something like:

```tsx
{(title: string | JSX.Element) => (
    <span
        id='back_button_icon'
        className='fa fa-1x fa-angle-left'
        title={title.toString()}
    />
)}
```

Reply:
Yup, that worked. Thanks!